src/invidious/routes/misc.cr

```diff
@@ -42,12 +42,17 @@ module Invidious::Routes::Misc
     referer = get_referer(env)
 
     instance_list = Invidious::Jobs::InstanceListRefreshJob::INSTANCES["INSTANCES"]
-    if instance_list.empty?
+    # Filter out the current instance
+    other_available_instances = instance_list.reject! { |_, domain| domain == CONFIG.domain }
```

Comment:
```suggestion
    other_available_instances = instance_list.reject { |_, domain| domain == CONFIG.domain }
```

`reject!` modifies itself instead of creating a new Tuple. `other_available_instances` isn't necessary unless you're using `reject`